component-library/src/app/components/component-preview/component-preview.component.scss

```diff
@@ -22,10 +22,9 @@ $selectors: "app-component-preview";
 
   .preview-wrapper {
     box-sizing: border-box;
-    padding: 0 64px;
```

Comment:
@mahsa-ameri I would like to forward [this comment](https://www.figma.com/file/TPwx1HcbXRCZeIDePI4Y4N?node-id=6613:232156&mode=design#514212097) to you.